src/app/tables/artifacts-table/artifacts-table.component.ts

```diff
@@ -70,16 +72,33 @@ export class ArtifactsTableComponent implements OnChanges {
       // Add a link mapping in the form of:
       // {
       //   734506: {
-      //     ID: 'https://koji.domain.local/koji/buildinfo?buildID=734506',
-      //     See Story: 'https://pipeline.domain.local/kojibuild/734506'
+      //     ID: {
+      //         Type: 'single',
+      //         Value: 'https://koji.domain.local/koji/buildinfo?buildID=734506',
+      //     See Story: {
+      //         Type: 'single',
+      //         Value: 'https://pipeline.domain.local/kojibuild/734506',
       //   }
       //   ...
       // }
       this.linkColumnMappings[formattedArtifact[this.uidColumn]] = {
-        [this.uidColumn]: externalUrlPipe.transform(artifact),
-        'See Story': this.getNodeStoryUrl(artifact)
+        [this.uidColumn]: linkColumnPipe.transform(externalUrlPipe.transform(artifact)),
```

Comment:
@sarah256 if you are using a pipe in the Component's functions, then there's no value in having it as a pipe over a standard function.

My suggestion is to keep the data as is without the pipe transforming it. Then in your template, you use the pipe to determine which column link you need to display.

Reply:
@mprahl The thing is the pipe makes it into the new `linkColumnMappings` format. So I would need a function then to make it into this format as well correct?